serverless/cmd/feeder/feed-to-github

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# feed-to-github is a wrapper around the serverless feeder command which does a 
```

Comment:
Should this have a `.sh` extension?

Reply:
Funnily enough it did have one originally, and then I read [this](https://google.github.io/styleguide/shellguide.html#file-extensions) so I removed it :)